client/src/plus/ai-help/index.tsx

```diff
@@ -257,12 +257,12 @@ export function AIHelpInner() {
                               </div>
                             );
                           },
-                          code: ({ inline, className, children, ...props }) => {
+                          code: ({ className, children, ...props }) => {
                             const match = /language-(\w+)/.exec(
                               className || ""
                             );
                             const lang = Prism.languages[match?.[1]];
-                            return !inline && lang ? (
+                            return lang ? (
```

Comment:
@LeoMcA I don't know what `inline` distinguishes, but didn't see a difference locally.

Reply:
Seems to distinguish between inline/block code blocks, however the presence/lack of a `language-foobar` class also seems to do the same. Even then, if I force Prism to operate on inline code blocks too (by setting `lang` to `Prism.languages["css"]` for instance), it seems to fail gracefully with inline blocks

Reply:
You're right, that makes sense. And what a relieve!